src/AutoBalance.cpp

```diff
@@ -48,8 +48,8 @@
 bool ABScriptMgr::OnBeforeModifyAttributes(Creature *creature, uint32 & instancePlayerCount) {
     bool ret=true;
     FOR_SCRIPTS_RET(ABModuleScript, itr, end, ret) // return true by default if not scripts
-        if (!itr->second->OnBeforeModifyAttributes(creature, instancePlayerCount))
-            ret=false; // we change ret value only when scripts return false
+    if (!itr->second->OnBeforeModifyAttributes(creature, instancePlayerCount))
+        ret=false; // we change ret value only when scripts return false
```

Comment:
add curly brackets

Reply:
there is a problem about indentation.
WRONG:
```C++
bool ABScriptMgr::OnBeforeModifyAttributes(Creature *creature, uint32 & instancePlayerCount) {
    bool ret=true;
    FOR_SCRIPTS_RET(ABModuleScript, itr, end, ret) // return true by default if not scripts
    if (!itr->second->OnBeforeModifyAttributes(creature, instancePlayerCount))
        {
        ret=false; // we change ret value only when scripts return false
        }
    return ret;
}
```
RIGHT:
```C++
bool ABScriptMgr::OnBeforeModifyAttributes(Creature *creature, uint32 & instancePlayerCount) {
    bool ret=true;
    FOR_SCRIPTS_RET(ABModuleScript, itr, end, ret) // return true by default if not scripts
    if (!itr->second->OnBeforeModifyAttributes(creature, instancePlayerCount)) {
        ret=false; // we change ret value only when scripts return false
    }
    return ret;
}
```

Reply:
@Helias Thanks. just updated.